pkg/sidecar/docker_reactor.go

```diff
@@ -55,7 +58,16 @@ func NewDockerReactor() (Reactor, error) {
 		return nil, err
 	}
 
+	client, err := sync.NewGenericClient(context.Background(), logging.S())
+	if err != nil {
+		return nil, err
+	}
```

Comment:
@Stebalien the reactor now creates a single sync client (i.e. 1 client per sidecar).